eland/cli/eland_import_hub_model.py

```diff
@@ -195,6 +196,17 @@ def get_es_client(cli_args, logger):
 
 def check_cluster_version(es_client, logger):
     es_info = es_client.info()
+
+    if (
+        "build_flavor" in es_info["version"]
+        and es_info["version"]["build_flavor"] == "serverless"
+    ):
```

Comment:
It's OK to assume `build_flavor` is set, it was here in the 7.x days too: https://www.elastic.co/guide/en/elasticsearch/reference/7.17/targz.html#_checking_that_elasticsearch_is_running

```suggestion
    if es_info["version"]["build_flavor"] == "serverless":
```

Reply:
Oh it failed running against a locally built and run Elasticsearch maybe it's not in dev builds. I'd prefer to keep this to be safe